src/util/graph.h

```diff
@@ -461,12 +464,21 @@ void grapht<N>::visit_reachable(node_indext src)
 template<class N>
 std::vector<typename N::node_indext>
 grapht<N>::get_reachable(node_indext src, bool forwards) const
+{
+  std::vector<node_indext> src_vector;
+  src_vector.push_back(src);
+
+  return get_reachable(src_vector, forwards);
+}
+
+template <class N>
+std::vector<typename N::node_indext>
+grapht<N>::get_reachable(std::vector<node_indext> src, bool forwards) const
 {
   std::vector<node_indext> result;
   std::vector<bool> visited(size(), false);
 
-  std::stack<node_indext, std::vector<node_indext>> s;
-  s.push(src);
+  std::stack<node_indext, std::vector<node_indext>> s(src);
```

Comment:
`src` is already passed as a copy, so there is no need for a second copy

Reply:
I find the current version better, it avoids potential problems if anybody uses the function in the future and does not expect the vector to change.